### ISSUE TITLE
[Documentation]: Peer dependency installation needs to be documented.

### ISSUE THREAD

**Issue by samwightt** [evoluhq/evolu#475]
Evolu's packages require various peer dependencies that need to be installed along with the Evolu package the user is consuming. If these aren't installed in a package manager like Yarn, then TypeScript and bundlers won't work correctly. Everything works right in Npm because npm hoists packages, but Yarn does not.

For React, the following need to be installed:

- @evolu/common
- @evolu/common-web
- @evolu/common-react
- @sqlite.org/sqlite-wasm
- @effect/platform
- @effect/schema (I think?)

For React Native, the following need to be installed:

- @evolu/common
- @evolu/common-react
- @evolu/common-web
- @evolu/react
- @effect/platform
- @effect/schema (I think?)
- @evolu/react-native
- expo-sqlite
- @sqlite.org/sqlite-wasm

Basically go through @evolu/react-native` and its deps and put all of the peer dependencies that are necessary into the docs. It would be great if the docs were updated with the packages needed to install.

**Comment by steida** [comment]
You are awesome, will update docs asap.

**Comment by steida** [comment]
Closing as this is no longer relevant—the new Evolu launches tomorrow. Feel free to reopen if it becomes relevant with the new Evolu.
